frontend/packages/console-dynamic-plugin-sdk/src/app/k8s/reducers/k8s.ts

```diff
@@ -0,0 +1,207 @@
+import { Map as ImmutableMap, fromJS } from 'immutable';
+import * as _ from 'lodash';
+import { K8sState } from '../../redux-types';
+import { ActionType, K8sAction } from '../actions/k8s';
+import { getK8sDataById } from './k8sSelector';
+
+const getQN: (obj) => string = (obj) => {
+  const { name, namespace } = obj.metadata;
+  // Name + namespace is not unique for PackageManifest resources, so include the catalog source.
+  // TODO: We should be able to remove this when the upstream OLM bug is fixed:
```

Comment:
From the meeting recording between @fhlavac and @invincibleJai we were looking to remove some of this technical debt. However when I follow the links to this bug and the resulting story in OLM, this doesn't appear to have been fixed yet.